src/main/java/io/quarkus/gizmo2/impl/AbstractMethodCreatorImpl.java

```diff
@@ -1,21 +1,20 @@
 package io.quarkus.gizmo2.impl;
 
-import static io.github.dmlloyd.classfile.extras.reflect.AccessFlag.ABSTRACT;
-import static io.github.dmlloyd.classfile.extras.reflect.AccessFlag.BRIDGE;
-import static io.github.dmlloyd.classfile.extras.reflect.AccessFlag.PROTECTED;
-import static io.github.dmlloyd.classfile.extras.reflect.AccessFlag.PUBLIC;
-import static io.github.dmlloyd.classfile.extras.reflect.AccessFlag.SYNCHRONIZED;
-import static io.github.dmlloyd.classfile.extras.reflect.AccessFlag.SYNTHETIC;
-import static io.github.dmlloyd.classfile.extras.reflect.AccessFlag.VARARGS;
+import static io.github.dmlloyd.classfile.ClassFile.*;
 
-import java.util.Set;
 import java.util.function.Consumer;
 
 import io.quarkus.gizmo2.creator.AbstractMethodCreator;
+import io.quarkus.gizmo2.creator.ModifierLocation;
 
 public final class AbstractMethodCreatorImpl extends MethodCreatorImpl implements AbstractMethodCreator {
     AbstractMethodCreatorImpl(final TypeCreatorImpl owner, final String name) {
-        super(owner, name, Set.of(ABSTRACT), Set.of(PUBLIC, PROTECTED, SYNCHRONIZED, SYNTHETIC, BRIDGE, ABSTRACT, VARARGS));
+        super(owner, name);
+        flags |= ACC_PUBLIC | ACC_ABSTRACT;
```

Comment:
`ACC_PUBLIC` is new here, is that intentional?

Reply:
Yeah, I thought package-private is not a very good default here. So I went a little bit back on my "don't change the defaults" promise I guess. But we should revisit it either way with #222.

Reply:
I agree we should revisit holistically, but changing the default just for abstract methods seems strange.

Reply:
Sure, my thinking was just that we didn't explicitly decide to make this package-private, but rather we forgot to have a default. And so the default was "default" just by chance.

But I can take this out if it's important.

Reply:
What I find important is we don't forget about this :-) No need to take it out right now.